sky/packages/sky/lib/widgets/transitions.dart

```diff
@@ -206,3 +206,38 @@ class SquashTransition extends TransitionBase {
     return new SizedBox(width: _maybe(width), height: _maybe(height), child: child);
   }
 }
+
+class BuilderTransition extends TransitionBase {
+  BuilderTransition({
+    Key key,
+    this.variables,
+    this.builder,
+    Duration duration,
+    AnimationPerformance performance,
+    Direction direction,
+    Function onDismissed,
+    Function onCompleted,
+    Widget child
+  }) : super(key: key,
+             duration: duration,
+             performance: performance,
+             direction: direction,
+             onDismissed: onDismissed,
+             onCompleted: onCompleted,
+             child: child);
+
+  List<AnimatedValue> variables;
+  Function builder;
+
+  void syncFields(BuilderTransition updated) {
```

Comment:
I agree.  After this CL lands, let's do a global rename of both the function name and parameter to something that's easier to understand.